### PR TITLE
etcdctl: snapshot status error for non-existent file

### PR DESCRIPTION
**Pull request by gyuho**

### CODE REVIEW

File: etcdctl/ctlv3/command/snapshot_command.go [hunk 1]
```diff
@@ -297,6 +297,12 @@ type dbstatus struct {
 }
 
 func dbStatus(p string) dbstatus {
+	if _, err := os.Stat(p); err != nil {
+		if os.IsNotExist(err) {
```
Comment (heyitsanthony) [inline etcdctl/ctlv3/command/snapshot_command.go]: I think it should just fail regardless when `err != nil`?

Comment (gyuho) [inline etcdctl/ctlv3/command/snapshot_command.go]: @heyitsanthony You are right. Just removed that check. PTAL.

**Comment by heyitsanthony** [comment]
```sh
  	db, err := bolt.Open(p, 0600, nil)
 	if err != nil {
```

doesn't handle this already, though?

**Comment by gyuho** [comment]
@heyitsanthony In my machine, `etcdctl snapshot status x.db` just prints out empty snapshot status without any error. Could you try in your machine?

**Comment by gyuho** [comment]
Like this...

```
./bin/etcdctl snapshot status zzzzzzzzzzz.db
+------+----------+------------+------------+
| HASH | REVISION | TOTAL KEYS | TOTAL SIZE |
+------+----------+------------+------------+
|    0 |        0 |          0 | 16 kB      |
+------+----------+------------+------------+
```

**Comment by heyitsanthony** [comment]
I think a `0400` instead of `0600` will fix it?

**Comment by heyitsanthony** [comment]
tried 0400 and I guess not... lgtm

**Comment by xiang90** [comment]
@heyitsanthony 

```
Open creates and opens a database at the given path. If the file does not exist then it will be created automatically. Passing in nil options will cause Bolt to open the database with the default options.
```

**Comment by gyuho** [comment]
yeah doesn't work either...

**Comment by gyuho** [comment]
merging in greens. thanks all!

**Pull request merged**